### ISSUE TITLE
[ENHANCEMENT] isSteamPowered functionality

### ISSUE THREAD

**Issue by boot2big** [DonBruce64/MinecraftTransportSimulator#1264]
![RobGrindes-shrug-143px](https://user-images.githubusercontent.com/53581325/180578504-eac19c72-4823-468c-9a5f-4245e89865da.png)

https://user-images.githubusercontent.com/53581325/180578810-15411ba8-8290-4f1a-9bb1-441c1f29e745.mp4

**Comment by DonBruce64** [comment]
If you didn't have that IV HUD up, I'd call that Fake News and say it's IR.  Cause I never thought I'd see a working IV train.  But a promise is a promise, so off to make steam engines I go!

**Comment by mikeve1205** [comment]
It had to be someone, and I made that train model to both spite and satiate someone's desire for trains in MTS, as well as to prove a point that we _already HAVE rail track physics, blasko_
It has the obligatory skin as well

Only issue at the moment is that roads are a little strange at first, but are somewhat easy to get used to

**Comment by DonBruce64** [comment]
Well, once I get a vehicle that has crates for coal in it... then I can poke that physics with a stick.  Cause somewhere, burnable things need to be stored.

**Comment by boot2big** [comment]
Here's a locomotive with crates for coal in it... :sans:
https://www.curseforge.com/minecraft/mc-mods/immersive-vehicles-vanity/
https://www.planetminecraft.com/mod/immersive-vehicles-vanity-mts-iv-pack/

**Comment by Colt44Magnum** [comment]
Provided picture of Locomotive and tender, though this will require vehicle to vehicle transfer of items 
![image](https://user-images.githubusercontent.com/51977107/213077912-67f0ad85-3a02-4a1e-b966-1678bbe1b001.png)

The E2 Class which what Thomas the Tank engine and I presume this Locomotive was based after actually has a small coal bunker on the vehicle itself that is at the back of the locomotive. Coal could be taken out of crates placed there

**Comment by DonBruce64** [comment]
Hey now, no double-dipping!  That's IVV content, not WARK!

**Comment by Colt44Magnum** [comment]
Yeah I know it's IVV. I got the image from Curseforge. What's with WARK?

**Comment by DonBruce64** [comment]
Now now, you know what that is.  It's on your pack's Curse page even!

**Comment by Colt44Magnum** [comment]
Guess I'm kinda confused still, yes WARK has some agriculture stuff in IV Farming, but that's not in RandallAg. But I don't understand what this had to do with WARK to begin with?

I posted the image because I'd thought It'd help the ticket move along, and gave an explanation on the E2 having a coal bunker at the end of it since you asked for a vehicle model
